### PR TITLE
Upgrade tap to 0.26

### PR DESCRIPTION
**Pull request by evankanderson**

### CODE REVIEW

File: kn@0.25.rb [hunk 1]
```diff
@@ -0,0 +1,29 @@
+require "fileutils"
+
+class Kn < Formula
```
Comment (rhuss) [inline kn@0.25.rb:3]:
```suggestion
class KnAT025 < Formula
```

This is required as every class must be unique across all formulas in an homebrew repo.

Comment (csantanapr) [reply kn@0.25.rb:3]: We need a test on PRs to catch this, same problem happened in homebrew for plugins

Comment (evankanderson) [reply kn@0.25.rb]: Updated, thank you!

Comment (evankanderson) [reply kn@0.25.rb]: Ugh, I missed a line in the release instructions: https://github.com/knative/release#homebrew-client

> In kn@${PREV_RELEASE}.rb replace class Kn with class KnAT${PREV_RELEASE_DIGITS}, e.g class KnAT019 for an previous release 0.19.

**Comment by csantanapr** [comment]
thanks @evankanderson 
/lgtm

**Pull request merged**